src/main/java/com/peachbros/letsmerge/user/model/domain/User.java

```diff
@@ -39,6 +55,24 @@ public User update(String email, String picture) {
         return this;
     }
 
+    public void addAssignInfo(AssignInfo assignInfo) {
+        this.assignedMissions.add(assignInfo);
+    }
+
+    public void cancelMission(Mission mission) {
+        AssignInfo assignInfo = assignedMissions.stream()
+                .filter(info -> Objects.equals(info.getMission().getId(), mission.getId()))
```

Comment:
```suggestion
                .filter(info -> info.matchId(mission.getId()))
```
위와 같이 수정하는게 더 낫지 않을까요?

Reply:
mission.getId() 대신 mission을 통째로 넘겼습니다!
아래 getAssignedMissions 로직도 비슷하게 변경했습니다 🙂